README.md

```diff
@@ -46,7 +46,7 @@ integrating Auryn with third-party libraries that use *container-interop*, such
 as [zend-expressive](https://github.com/zendframework/zend-expressive).
 
 The implementation in this repository takes a small amount of liberty with this
-passage from [Section 1.1](https://github.com/container-interop/container-interop/blob/master/docs/ContainerInterface.md#11-basics)
+passage from [Section 1.1](https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-11-container.md#11-basics)
```

Comment:
Hi @elazar , 

I think probably we can get rid of this section, for the current get and has doesn't have the problem mentioned as in container-interop. Let me know what you think.

Reply:
@harikt I'm not sure I follow. The same quote exists in both documents. Can you elaborate?

Reply:
Hi @elazar , 

The delegate lookup feature is not currently in `PSR-11` , but is only present in `container-interop`. 

See discussion over https://github.com/laravel/internals/issues/803#issue-259409277 .

Let me know if that helps! .

Thank you.

Reply:
@harikt I think I see what you're getting at. I'm open to suggestions for amending or removing the cited passage.